### PR TITLE
Fix BitPlatform typos in all files

### LINKED ISSUE #5426: Typo in `bit platform` usages all over the repo
Typo in `bit platform` usages all over the repo:

Multiple places in the repo we have used this term with typos such as `BitPlatform` or `bitplatform` which should be `bit platform`.

### PR DESCRIPTION
**Pull request by taha-ghadirian**
Closes #5426

### CODE REVIEW

File: src/BlazorUI/Demo/Client/Core/Pages/TermsPage.razor [hunk 1]
```diff
@@ -29,21 +29,21 @@
         <section class="sub-section">
             <h2 class="sub-section-title">License Grant</h2>
             <div class="section-desc">
-                BitPlatform hereby grants the user a non-exclusive, non-transferable license to use the Software, solely for the purpose of evaluating and testing the features of the bit BlazorUI Demo during the term of this EULA.
+                Bit Platform hereby grants the user a non-exclusive, non-transferable license to use the Software, solely for the purpose of evaluating and testing the features of the bit BlazorUI Demo during the term of this EULA.
```
Comment (msynk) [inline src/BlazorUI/Demo/Client/Core/Pages/TermsPage.razor:32]: Lowercase all the way plz

File: src/BlazorUI/Demo/Client/Core/Pages/TermsPage.razor [hunk 2]
```diff
@@ -29,21 +29,21 @@
...
             </div>
         </section>
 
         <section class="sub-section">
             <h2 class="sub-section-title">Ownership</h2>
             <div class="section-desc">
-                BitPlatform retains all rights, title, and interest in and to the Software, including any and all intellectual property rights.
+                Bit Platform retains all rights, title, and interest in and to the Software, including any and all intellectual property rights.
```
Comment (msynk) [inline src/BlazorUI/Demo/Client/Core/Pages/TermsPage.razor:39]: Lowercase all the way plz

File: src/BlazorUI/Demo/Client/Core/Pages/TermsPage.razor [hunk 3]
```diff
@@ -29,21 +29,21 @@
...
 
         <section class="sub-section">
             <h2 class="sub-section-title">User Data</h2>
             <div class="section-desc">
-                BitPlatform may collect user data, including name, surname, photo, date of birth, and gender. Providing this information is optional, and BitPlatform will only use it for the purposes of the bit BlazorUI Demo. If the user requests, BitPlatform will delete this data within one day, using one of the contact methods provided at the end of this EULA. In addition, the user is required to provide an email address for account verification purposes, but BitPlatform will not use this email for any other purposes.
+                Bit platform may collect user data, including name, surname, photo, date of birth, and gender. Providing this information is optional, and bit platform will only use it for the purposes of the bit BlazorUI Demo. If the user requests, Bit platform will delete this data within one day, using one of the contact methods provided at the end of this EULA. In addition, the user is required to provide an email address for account verification purposes, but Bit platform will not use this email for any other purposes.
```
Comment (msynk) [inline src/BlazorUI/Demo/Client/Core/Pages/TermsPage.razor:46]: Lowercase all the way plz

File: src/BlazorUI/Demo/Client/Core/Pages/TermsPage.razor [hunk 4]
```diff
@@ -102,7 +102,7 @@
             <h1 class="main-section-title">Privacy Policy</h1>
             <div class="section-desc">
                 <ul>
-                    <li>BitPlatform is committed to protecting the privacy of user data.</li>
+                    <li>Bit platform is committed to protecting the privacy of user data.</li>
```
Comment (msynk) [inline src/BlazorUI/Demo/Client/Core/Pages/TermsPage.razor:105]: Lowercase all the way plz

File: src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Pages/TermsPage.razor [hunk 5]
```diff
@@ -27,28 +27,28 @@
         <section class="sub-section">
             <h2 class="sub-section-title">License Grant</h2>
             <div class="section-desc">
-                BitPlatform hereby grants the user a non-exclusive, non-transferable license to use the Software, solely for the purpose of evaluating and testing the features of the AdminPanel Demo Version during the term of this EULA.
+                Bit platform hereby grants the user a non-exclusive, non-transferable license to use the Software, solely for the purpose of evaluating and testing the features of the AdminPanel Demo Version during the term of this EULA.
```
Comment (msynk) [inline src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Pages/TermsPage.razor:30]: Lowercase all the way plz

File: src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Pages/TermsPage.razor [hunk 6]
```diff
@@ -27,28 +27,28 @@
...
             </div>
         </section>
 
         <section class="sub-section">
             <h2 class="sub-section-title">Ownership</h2>
             <div class="section-desc">
-                BitPlatform retains all rights, title, and interest in and to the Software, including any and all intellectual property rights.
+                Bit platform retains all rights, title, and interest in and to the Software, including any and all intellectual property rights.
```
Comment (msynk) [inline src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Pages/TermsPage.razor:37]: Lowercase all the way plz

File: src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Pages/TermsPage.razor [hunk 7]
```diff
@@ -27,28 +27,28 @@
...
             <h2 class="sub-section-title">User Data</h2>
             <div class="section-desc">
-                BitPlatform may collect user data, including name, surname, photo, date of birth, and gender. Providing this information is optional, and BitPlatform will only use it for the purposes of the AdminPanel Demo Version. If the user requests, BitPlatform will delete this data within one day, using one of the contact methods provided at the end of this EULA. In addition, the user is required to provide an email address for account verification purposes, but BitPlatform will not use this email for any other purposes.
+                Bit platform may collect user data, including name, surname, photo, date of birth, and gender. Providing this information is optional, and bit platform will only use it for the purposes of the AdminPanel Demo Version. If the user requests, bit platform will delete this data within one day, using one of the contact methods provided at the end of this EULA. In addition, the user is required to provide an email address for account verification purposes, but bit platform will not use this email for any other purposes.
```
Comment (msynk) [inline src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Pages/TermsPage.razor:44]: Lowercase all the way plz

File: src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Pages/TermsPage.razor [hunk 8]
```diff
@@ -27,28 +27,28 @@
...
             </div>
         </section>
 
         <section class="sub-section">
             <h2 class="sub-section-title">Analytics and Cookies</h2>
             <div class="section-desc">
-                BitPlatform does not use any Analytics tools for measuring website traffic and only uses strictly necessary cookies.
+                Bit platform does not use any Analytics tools for measuring website traffic and only uses strictly necessary cookies.
```
Comment (msynk) [inline src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Pages/TermsPage.razor:51]: Lowercase all the way plz

File: src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Pages/TermsPage.razor [hunk 9]
```diff
@@ -62,28 +62,28 @@
         <section class="sub-section">
             <h2 class="sub-section-title">No Warranty</h2>
             <div class="section-desc">
-                The Software is provided "AS IS" and without warranty of any kind, express or implied, including but not limited to the warranties of merchantability, fitness for a particular purpose, and non-infringement. BitPlatform does not warrant that the Software will be error-free or that its use will be uninterrupted.
+                The Software is provided "AS IS" and without warranty of any kind, express or implied, including but not limited to the warranties of merchantability, fitness for a particular purpose, and non-infringement. Bit platform does not warrant that the Software will be error-free or that its use will be uninterrupted.
```
Comment (msynk) [inline src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Pages/TermsPage.razor:65]: Lowercase all the way plz

File: src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Pages/TermsPage.razor [hunk 10]
```diff
@@ -62,28 +62,28 @@
...
             </div>
         </section>
 
         <section class="sub-section">
             <h2 class="sub-section-title">Limitation of Liability</h2>
             <div class="section-desc">
-                BitPlatform shall not be liable for any damages whatsoever arising out of the use of or inability to use the Software, including but not limited to direct, indirect, incidental, special, or consequential damages, even if BitPlatform has been advised of the possibility of such damages.
+                Bit platform shall not be liable for any damages whatsoever arising out of the use of or inability to use the Software, including but not limited to direct, indirect, incidental, special, or consequential damages, even if bit platform has been advised of the possibility of such damages.
```
Comment (msynk) [inline src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Pages/TermsPage.razor:72]: Lowercase all the way plz

File: src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Pages/TermsPage.razor [hunk 11]
```diff
@@ -62,28 +62,28 @@
...
             </div>
         </section>
 
         <section class="sub-section">
             <h2 class="sub-section-title">Termination</h2>
             <div class="section-desc">
-                This EULA shall remain in effect until 2025-12-31. BitPlatform reserves the right to terminate this EULA at any time without notice. Upon termination, the user shall immediately cease all use of the Software and destroy all copies of the Software in its possession.
+                This EULA shall remain in effect until 2025-12-31. Bit platform reserves the right to terminate this EULA at any time without notice. Upon termination, the user shall immediately cease all use of the Software and destroy all copies of the Software in its possession.
```
Comment (msynk) [inline src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Pages/TermsPage.razor:79]: Lowercase all the way plz

File: src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Pages/TermsPage.razor [hunk 12]
```diff
@@ -62,28 +62,28 @@
...
             </div>
         </section>
 
         <section class="sub-section">
             <h2 class="sub-section-title">Contact Information</h2>
             <div class="section-desc">
-                If you have any questions or concerns regarding this EULA or the Privacy Policy, or if you would like to contact BitPlatform for any reason, please email <BitLink Href="mailto:info@bitplatform.dev">info@bitplatform.dev</BitLink> or call <BitLink Href="tel:+31684207362">+31684207362</BitLink>, or visit our website at <BitLink Href="https://bitplatform.dev/contact-us" Target="_blank">our website</BitLink> or visit us at Sixmastraat 15, 8932 PA Leeuwarden, Netherlands.
+                If you have any questions or concerns regarding this EULA or the Privacy Policy, or if you would like to contact Bit platform for any reason, please email <BitLink Href="mailto:info@bitplatform.dev">info@bitplatform.dev</BitLink> or call <BitLink Href="tel:+31684207362">+31684207362</BitLink>, or visit our website at <BitLink Href="https://bitplatform.dev/contact-us" Target="_blank">our website</BitLink> or visit us at Sixmastraat 15, 8932 PA Leeuwarden, Netherlands.
```
Comment (msynk) [inline src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Pages/TermsPage.razor:86]: Lowercase all the way plz

File: src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Pages/TermsPage.razor [hunk 13]
```diff
@@ -92,7 +92,7 @@
             <h1 class="main-section-title">Privacy Policy</h1>
             <div class="section-desc">
                 <ul>
-                    <li>BitPlatform is committed to protecting the privacy of user data.</li>
+                    <li>Bit platform is committed to protecting the privacy of user data.</li>
```
Comment (msynk) [inline src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Pages/TermsPage.razor:95]: Lowercase all the way plz

File: src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Core/Pages/TermsPage.razor [hunk 14]
```diff
@@ -9,7 +9,7 @@
         <section class="main-section">
             <h1 class="main-section-title">TodoTemplate license</h1>
             <div class="section-desc">
-                This EULA and Privacy Policy are written for individuals who use the TodoTemplate Demo Version through the <BitLink Href="https://todo.bitplatform.dev" Target="_blank">todo.bitplatform.dev</BitLink> website, or the published version of this app on the BitPlatform's Google Play, Apple Store, and Microsoft Store accounts.
+                This EULA and Privacy Policy are written for individuals who use the TodoTemplate Demo Version through the <BitLink Href="https://todo.bitplatform.dev" Target="_blank">todo.bitplatform.dev</BitLink> website, or the published version of this app on the Bit platform's Google Play, Apple Store, and Microsoft Store accounts.
```
Comment (msynk) [inline src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Core/Pages/TermsPage.razor:12]: Lowercase all the way plz

File: src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Core/Pages/TermsPage.razor [hunk 15]
```diff
@@ -18,7 +18,7 @@
         <section class="sub-section">
             <h2 class="sub-section-title">End-User License Agreement (EULA) for TodoTemplate Demo Version</h2>
             <div class="section-desc">
-                This End-User License Agreement (EULA) is a legal agreement between you (the "user") and BitPlatform, located at Sixmastraat 15, 8932 PA Leeuwarden, Netherlands, with phone number <BitLink Href="tel:+31684207362">+31684207362</BitLink>, for the use of TodoTemplate Demo Version (the "Software").
+                This End-User License Agreement (EULA) is a legal agreement between you (the "user") and Bit platform, located at Sixmastraat 15, 8932 PA Leeuwarden, Netherlands, with phone number <BitLink Href="tel:+31684207362">+31684207362</BitLink>, for the use of TodoTemplate Demo Version (the "Software").
```
Comment (msynk) [inline src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Core/Pages/TermsPage.razor:21]: Lowercase all the way plz

File: src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Core/Pages/TermsPage.razor [hunk 16]
```diff
@@ -27,28 +27,28 @@
         <section class="sub-section">
             <h2 class="sub-section-title">License Grant</h2>
             <div class="section-desc">
-                BitPlatform hereby grants the user a non-exclusive, non-transferable license to use the Software, solely for the purpose of evaluating and testing the features of the TodoTemplate Demo Version during the term of this EULA.
+                Bit platform hereby grants the user a non-exclusive, non-transferable license to use the Software, solely for the purpose of evaluating and testing the features of the TodoTemplate Demo Version during the term of this EULA.
```
Comment (msynk) [inline src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Core/Pages/TermsPage.razor:30]: Lowercase all the way plz

File: src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Core/Pages/TermsPage.razor [hunk 17]
```diff
@@ -27,28 +27,28 @@
...
             </div>
         </section>
 
         <section class="sub-section">
             <h2 class="sub-section-title">Ownership</h2>
             <div class="section-desc">
-                BitPlatform retains all rights, title, and interest in and to the Software, including any and all intellectual property rights.
+                Bit platform retains all rights, title, and interest in and to the Software, including any and all intellectual property rights.
```
Comment (msynk) [inline src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Core/Pages/TermsPage.razor:37]: Lowercase all the way plz

File: src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Core/Pages/TermsPage.razor [hunk 18]
```diff
@@ -27,28 +27,28 @@
...
             <h2 class="sub-section-title">User Data</h2>
             <div class="section-desc">
-                BitPlatform may collect user data, including name, surname, photo, date of birth, and gender. Providing this information is optional, and BitPlatform will only use it for the purposes of the TodoTemplate Demo Version. If the user requests, BitPlatform will delete this data within one day, using one of the contact methods provided at the end of this EULA. In addition, the user is required to provide an email address for account verification purposes, but BitPlatform will not use this email for any other purposes.
+                Bit platform may collect user data, including name, surname, photo, date of birth, and gender. Providing this information is optional, and Bit platform will only use it for the purposes of the TodoTemplate Demo Version. If the user requests, Bit platform will delete this data within one day, using one of the contact methods provided at the end of this EULA. In addition, the user is required to provide an email address for account verification purposes, but Bit platform will not use this email for any other purposes.
```
Comment (msynk) [inline src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Core/Pages/TermsPage.razor:44]: Lowercase all the way plz

File: src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Core/Pages/TermsPage.razor [hunk 19]
```diff
@@ -27,28 +27,28 @@
...
             </div>
         </section>
 
         <section class="sub-section">
             <h2 class="sub-section-title">Analytics and Cookies</h2>
             <div class="section-desc">
-                BitPlatform does not use any Analytics tools for measuring website traffic and only uses strictly necessary cookies.
+                Bit platform does not use any Analytics tools for measuring website traffic and only uses strictly necessary cookies.
```
Comment (msynk) [inline src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Core/Pages/TermsPage.razor:51]: Lowercase all the way plz

File: src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Core/Pages/TermsPage.razor [hunk 20]
```diff
@@ -62,28 +62,28 @@
         <section class="sub-section">
             <h2 class="sub-section-title">No Warranty</h2>
             <div class="section-desc">
-                The Software is provided "AS IS" and without warranty of any kind, express or implied, including but not limited to the warranties of merchantability, fitness for a particular purpose, and non-infringement. BitPlatform does not warrant that the Software will be error-free or that its use will be uninterrupted.
+                The Software is provided "AS IS" and without warranty of any kind, express or implied, including but not limited to the warranties of merchantability, fitness for a particular purpose, and non-infringement. Bit platform does not warrant that the Software will be error-free or that its use will be uninterrupted.
```
Comment (msynk) [inline src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Core/Pages/TermsPage.razor:65]: Lowercase all the way plz

File: src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Core/Pages/TermsPage.razor [hunk 21]
```diff
@@ -62,28 +62,28 @@
...
             </div>
         </section>
 
         <section class="sub-section">
             <h2 class="sub-section-title">Limitation of Liability</h2>
             <div class="section-desc">
-                BitPlatform shall not be liable for any damages whatsoever arising out of the use of or inability to use the Software, including but not limited to direct, indirect, incidental, special, or consequential damages, even if BitPlatform has been advised of the possibility of such damages.
+                Bit platform shall not be liable for any damages whatsoever arising out of the use of or inability to use the Software, including but not limited to direct, indirect, incidental, special, or consequential damages, even if Bit platform has been advised of the possibility of such damages.
```
Comment (msynk) [inline src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Core/Pages/TermsPage.razor:72]: Lowercase all the way plz

**Review by msynk — CHANGES_REQUESTED** [review]

**Comment by taha-ghadirian** [comment]
Requested changes applied. 

However, As a suggestion I guess starting sentences with uppercase letters, makes the code and documents easier to read and understand.

**Comment by msynk** [comment]
@taha-ghadirian Thanks.
I understand your suggestion, but in this case, since the `bit platform` is a special name, its capitalization should not be changed anywhere.

File: src/BlazorUI/Demo/Client/Core/Pages/TermsPage.razor [hunk 22]
```diff
@@ -29,21 +29,21 @@
...
 
         <section class="sub-section">
             <h2 class="sub-section-title">User Data</h2>
             <div class="section-desc">
-                BitPlatform may collect user data, including name, surname, photo, date of birth, and gender. Providing this information is optional, and BitPlatform will only use it for the purposes of the bit BlazorUI Demo. If the user requests, BitPlatform will delete this data within one day, using one of the contact methods provided at the end of this EULA. In addition, the user is required to provide an email address for account verification purposes, but BitPlatform will not use this email for any other purposes.
+                bit platform may collect user data, including name, surname, photo, date of birth, and gender. Providing this information is optional, and bit platform will only use it for the purposes of the bit BlazorUI Demo. If the user requests, Bit platform will delete this data within one day, using one of the contact methods provided at the end of this EULA. In addition, the user is required to provide an email address for account verification purposes, but bit platform will not use this email for any other purposes.
```
Comment (msynk) [inline src/BlazorUI/Demo/Client/Core/Pages/TermsPage.razor:46]: you missed one here

**Review by msynk — CHANGES_REQUESTED** [review]

**Review by msynk — APPROVED** [review]

**Pull request merged**